spec/features/user_edit_his_review_spec.rb

```diff
@@ -18,7 +18,7 @@
       length: '6',
```

Comment:
Please rename this file to use gender neutral terms, or just `user_edits_review`